macros/src/main/scala/io/methvin/play/autoconfig/AutoConfigImpl.scala

```diff
@@ -48,7 +48,7 @@ private[autoconfig] class AutoConfigImpl(val c: blackbox.Context) {
       params.map { p =>
         val name = p.annotations.collectFirst {
           case ann if ann.tree.tpe =:= typeOf[ConfigName] =>
-            ann.tree.children.last.toString
+            ann.tree.children.last.toString.replace("\"", "")
```

Comment:
Maybe something like this would be slightly less ugly:

```scala
ann.tree.children.last.asInstanceOf[Literal].value.value.asInstanceOf[String]
```

Reply:
or how about

```scala
ann.tree.children.collectFirst { case Literal(Constant(str: String)) => str }.get
```